doc/helper_scripts/show_fields.py

```diff
@@ -37,7 +37,7 @@ def _strip_ftype(field):
 
 
 np.random.seed(int(0x4D3D3D3))
-units = [base_ds._get_field_info(*f).units for f in fields]
+units = [base_ds._get_field_info(f).units for f in fields]
```

Comment:
hm, this is because it's always a tuple now right?

Reply:
More exactly, it's because we don't have a second positional argument anymore. Passing a full tuple as the first argument already worked on main, I just made it the only way to do it.

Reply:
I have no idea, probably no good reason, but this feels just slightly out of scope here. Do you mind if I leave the line as is ?